### ISSUE TITLE
Test coverage toolbar quickpick - unintuitive dismissal using mouse

### ISSUE THREAD

**Issue by gjsjohnmurray** [microsoft/vscode#212866]
Type: <b>Bug</b>

1. Use a testprovider extension that implements coverage (I used VS Code's own).
2. Perform a coverage run.
3. When the coverage toolbar appears above the document click the test selector (which incidentally would benefit from having a tooltip added).

A top-centre quickpick offers me the choices, and uses `ignoreFocusOut` so I can explore elsewhere before deciding. But to cancel the dialog because I changed my mind about needing it I must either reach for the keyboard and press Esc or with the mouse I must click on the already-highlighted current selection.

This is another case where the automatic `x` button proposed in #99081 for `ignoreFocusOut` quickpicks would be beneficial.

VS Code version: Code - Insiders 1.90.0-insider (0cd8d6ac480bc5b4e2a2d7f1eb87eaff9f7d47e1, 2024-05-16T05:49:53.998Z)
OS version: Windows_NT x64 10.0.22631
Modes:

**Comment by connor4312** [comment]
This is how quickpicks works in general, giving it to @TylerLeonhardt to potentially duplicate

**Comment by TylerLeonhardt** [comment]
Yeah this is either a duplicate of https://github.com/microsoft/vscode/issues/99081

Or, I ask, does the test coverage thing need to use ignoreFocusOut?

**Comment by connor4312** [comment]
I put it there because I figured people might want to look around their code or at other things while deciding what test they want to filter to. I don't feel super strongly in this case though.

**Comment by TylerLeonhardt** [comment]
They can scroll the editor without losing focus from the quick pick, right? Also it's an easy click away to bring it back.

I feel like this doesn't need ignoreFocusOut... but I haven't used it yet. Just my gut feeling.

@gjsjohnmurray thoughts?

**Comment by gjsjohnmurray** [comment]
I think it'll be OK to drop `ignoreFocusOut` for this one. But I wish we'd get the generic solution done.
